src/components/course-periods-nav.cjsx

```diff
@@ -57,7 +58,14 @@ CoursePeriodsNav = React.createClass
     @setState(active: key)
 
   renderPeriod: (period, key) ->
-    <BS.NavItem eventKey={key} key="period-nav-#{period.id}">{period.name}</BS.NavItem>
+    {plan} = @props
+    isTrouble = _.pluck(plan.stats, 'is_trouble')
+    <BS.NavItem 
+      className={'is-trouble' if _.contains(isTrouble, true)}
```

Comment:
should this be `className={'is-trouble' if period.is_trouble}`?

If so, then is the `plan` prop used anymore?